docs/examples/decisiontree.jl

```diff
@@ -0,0 +1,161 @@
+#=
+# Visual representation of a decision tree
+
+In this example we are going to plot a decision tree of type [`DecisionTree.jl`](https://github.com/JuliaAI/DecisionTree.jl) using the
+Bucheim Layout from [`NetworkLayout.jl`](https://github.com/JuliaGraphs/NetworkLayout.jl).
+=#
+using CairoMakie
+using Graphs
+using GraphMakie
+using MLJ
+using NetworkLayout
+
```

Comment:
`using DecisionTree`
is missing here (has `load_data`)

Reply:
... then `DecisionTree` should go in the `docs/Project.tml` as well

BTW: sorry for ignoring you on Discord again, I don't get any notifications there. I think its allways easier to reach me via discourse PM, mail or on slack. But to answer the qustion: The recipe style isfine, those are examples anyway so I think it is nice to showcase various different ways of working with Makie/GraphMakie.